packages/botonic-plugin-directus/src/cms/contents.ts

```diff
@@ -0,0 +1,50 @@
+export interface CommonFields {
+  id: string
+  name: string
+  shortText?: string
+  keywords?: string[]
+  followup?: Text | Image
+  customFields?: Record<string, unknown>
+}
+
+export abstract class Content {
+  readonly common: CommonFields
+  constructor(common: CommonFields) {
+    this.common = common
+  }
+  get name(): string {
+    return this.common.name
+  }
+
+  get id(): string {
+    return this.common.id
+  }
+}
+
+export class Text extends Content {
+  readonly text: string
+  readonly buttons?: Button[]
+  constructor(opt: { common: CommonFields; text: string; buttons?: Button[] }) {
+    super(opt.common)
+    this.text = opt.text
+    this.buttons = opt.buttons
+  }
+}
+
+export class Button extends Content {
+  readonly text: string
+  readonly target: string
+  constructor(opt: { common: CommonFields; text: string; target: string }) {
+    super(opt.common)
+    this.text = opt.text
+    this.target = opt.target
+  }
+}
+
+export class Image extends Content {
+  readonly image: string
```

Comment:
Should `src` or `url` be a better name for this attribute than `image`? This attribute is representing the URL/path of the `Image`, right?